### PR TITLE
Fixes #22554: Typo in windows msiexec command in documentation

### PR DESCRIPTION
**Pull request by ElaadF**
https://issues.rudder.io/issues/22554

### CODE REVIEW

File: src/reference/modules/installation/pages/agent/windows.adoc [hunk 1]
```diff
@@ -57,7 +57,15 @@ By default, if no user is given at install time, the scheduled tasks will be run
...
 In case of error, logs are available in `C:\Users\vagrant\AppData\Local\Temp`. The file will be named MSIxxx.log, sort them by date to find the last one.
 
-To get verbose logs, use msiexec's option `/l*v MyLogFile.txt`. You can then find errors by looking for `value 3`.
+To get verbose logs, use msiexec's option `/L*V MyLogFile.txt`. You can then find errors by looking for `value 3`.
+
+[INFO]
+
+====
+
+The MyLogFile.txt could be interpreted as a binary file from some editor, we recommend to use `vim` to read the log without corrupted text.
```
[the remark this answers is not in the document]
Comment (amousset) [reply src/reference/modules/installation/pages/agent/windows.adoc:66]: Which which editors did got have a problem?

[the remark this answers is not in the document]
Comment (ElaadF) [reply src/reference/modules/installation/pages/agent/windows.adoc:66]: atom at least

File: src/reference/modules/installation/pages/agent/windows.adoc [hunk 2]
```diff
@@ -57,7 +57,15 @@ By default, if no user is given at install time, the scheduled tasks will be run
 
 In case of error, logs are available in `C:\Users\vagrant\AppData\Local\Temp`. The file will be named MSIxxx.log, sort them by date to find the last one.
 
-To get verbose logs, use msiexec's option `/l*v MyLogFile.txt`. You can then find errors by looking for `value 3`.
+To get verbose logs, use msiexec's option `/L*V MyLogFile.txt`. You can then find errors by looking for `value 3`.
```
Comment (ElaadF) [inline src/reference/modules/installation/pages/agent/windows.adoc:60]: https://learn.microsoft.com/fr-fr/windows-server/administration/windows-commands/msiexec#examples-3

**Comment by ElaadF** [comment]
PR updated with a new commit

**Comment by ElaadF** [comment]
Commit modified

**Pull request merged**